rfc.xml

```diff
@@ -435,34 +450,55 @@
 			</t>
 			<!-- TODO: clunky. We could clean this. ~MM -->
 		</section>
-		<section anchor="securityconsiderations" title="Security Considerations">
-			<section anchor="securitybenefits" title="Security Benefits">
+		<section anchor="securitybenefits" title="Security Benefits">
+			<t>
+				The keylist subscription functionality defined in this
+				document provides a number of security benefits, including:
+			</t>
+			<t>
+				<list style="symbols">
+					<t>
+						The ability for new keys to be quickly distributed
+						across an organization.
+					</t>
+					<t>
+						Removing the complexity of key distribution from
+						end users, allowing them to focus on the content of
+						their communications rather than on key management.
+					</t>
+					<t>
+						The ability for an organization to prevent the
+						spread of falsely attributed keys by centralizing
+						the public key discovery process within their
+						organization without centralized public key hosting.
+					</t>
+					<!-- TODO: others? ~MM -->
+				</list>
+			</t>
+		</section>
+		<section anchor="comparison" title="Relation to Other Technologies">
+			<section anchor="wkd" title="Web Key Directories">
 				<t>
-					The keylist subscription functionality defined in this
-					document provide a number of security benefits, including:
+					Unlike Web Key Directories, keylists are not domain specific. A keylist
+					might contain public key fingerprints for email addresses across several
+					different domains. Moreover, keylists only provide references to public
+					keys by way of fingerprints; Web Key Directories provide the public keys
+					themselves.
 				</t>
+			</section>
+			<section anchor="dns" title="OPENPGPKEY DNS Records">
```

Comment:
I'll mention one interesting thing that may be relevant (or not). Another I-D - [security.txt uses URIs to refer to PGP keys](https://tools.ietf.org/html/draft-foudil-securitytxt-05#section-3.5.4). That gives them ability to cover in one consistent way a) OPENPGPKEY DNS based keys, b) V4 fingerprints and even c) HTTPS links to keys.

Reply:
Hmm, that's interesting. Might be something we include in v05? @micahflee would allowing URIs seriously complicate things in the implementation? If not, we might consider including it. It'd resolve a lot of the ambiguity around fingerprints, which kinds to allow, and give more flexibility in key sourcing.

I'll create an issue for this.

Reply:
See #12